Login_library/src/main/java/com/example/Login_library/controller/AuthenticationController.java

```diff
@@ -39,4 +43,21 @@ public ResponseEntity<AuthenticationResponse> register(@RequestBody Authenticati
 
         return ResponseEntity.ok(authenticationService.authenticate(request));
     }
+
+    @GetMapping()
+    public List<User> getEveryUsers(User user) {
+        return userService.getallUsers();
+    }
+
+    @GetMapping("{id}")
+    public User getUsersById(@PathVariable("id") long id) {
+        return userService.getUserById(id);
+    }
+
+    @DeleteMapping("{id}")
+    public void getDeleteById(@PathVariable("id") long id) {
+         userService.deleteUser(userRepository.findById(id).orElseThrow(()-> new RuntimeException("no user with this id") ));
```

Comment:
userService call deleteUser and deleteUser return userRepository(User)
go to the userService and return userService.deleteUser(userRepository.findById(id).orElseThrow(()-> new RuntimeException("no user with this id") ));